Python/errors.c

```diff
@@ -1610,8 +1612,8 @@ _PyErr_WriteUnraisableMsg(const char *err_msg_str, PyObject *obj)
         }
     }
 
-    if (err_msg_str != NULL) {
-        err_msg = PyUnicode_FromFormat("Exception ignored %s", err_msg_str);
+    if (format != NULL) {
+        err_msg = PyUnicode_FromFormatV(format, va);
         if (err_msg == NULL) {
             PyErr_Clear();
```

Comment:
I don't think that it's the best behavior. I would prefer to ``goto error``. What do you think?

Maybe ``PyUnicode_FromString("Exception ignored in sys.unraisablehook")`` can become a ``_Py_DECLARE_STR(exc_ignored_msg, "...")`` + ``_Py_STR(exc_ignored_msg)`` static immortal string to avoid error 3 while handling error 2 while handling error 1.